c2corg_ui/static/js/geomdownload.js

```diff
@@ -56,11 +66,21 @@ app.TrackDownloadController = function(ngeoDownload, mapFeatureCollection) {
  * @param {string} mimetype
  * @private
  */
-app.TrackDownloadController.prototype.downloadFeatures_ = function(
+app.GeomDownloadController.prototype.downloadFeatures_ = function(
   format, extension, mimetype) {
   var geojson = new ol.format.GeoJSON();
   var features = geojson.readFeatures(this.featureCollection_);
-  features = features.filter(app.utils.isLineFeature);
+  switch (this.featureType) {
+    case 'Line':
+      features = features.filter(app.utils.isLineFeature);
+      break;
+    case 'Point':
+      // export only the waypoint itself, not its associated features
+      features = [features[0]];
```

Comment:
Yes, in detail view pages, ``features`` is set with ``mapFeatureCollection``:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/waypoint/view.html#L40-L68

Which is useful to display the associated WP on the map as well.